asapdiscovery-alchemy/asapdiscovery/alchemy/utils.py

```diff
@@ -103,8 +136,16 @@ def action_network(
                 self._client.create_tasks(tf_sk, count=planned_network.n_repeats + 1)
             )
 
+        # set the network weight based on the found network weights that are currently running while making sure the weight doesn't fall outside 0-1
+        if prioritize is None:
+            weight = 0.5
+        elif prioritize:
```

Comment:
This will accepty any `truthy` value eg "a", may want to check `if prioritize is True` 

In fact I might restructure this a bit so that the branches are 


```python

if prioritize is True: 
    ... # prioritize
elif prioritize is False:
    ... #deprioritize 
 else:
     # use 0.5 
 ```
 
 That way the base case is not de-prioritizing.

Reply:
Would it make our life easier to just use priority categories like `normal=0.5`, `medium=0.75` and `high=1`?  It would save us from having to get all other actioned weights when setting the priority as well which should make it faster. We can expose a way to change the priority of a network via the CLI as well.

Reply:
Would it make our life easier to just use priority categories like `normal=0.5`, `medium=0.75` and `high=1`?  It would save us from having to get all other actioned weights when setting the priority as well which should make it faster. We can expose a way to change the priority of a network via the CLI as well.

Reply:
@jthorton that could still quite easily lead to a situation where we have a network running at `1`, which would block us from submitting any networks of even higher priority. Having it bump by small increments like in the above would essentially prevent that situation from ever happening

Reply:
Good point I have kept the current method and changed it slightly to ensure no networks get a weight of 0.0 as this stops all compute on them.